packages/minifiers/minifiers-tests.js

```diff
@@ -13,9 +13,7 @@ Tinytest.add("minifiers - simple css minification", function (test) {
   t('p > *.my-class { color: #fff; }', 'p>.my-class{color:#fff}', 'removing optional whitespace around ">" in selector');
   t('p +  *.my-class { color: #fff; }', 'p+.my-class{color:#fff}', 'removing optional whitespace around "+" in selector');
   // XXX url parsing is difficult to support at the moment
-  t('a {\n\
-  font:12px \'Helvetica\',"Arial",\'Nautica\';\n\
-  background:url("/some/nice/picture.png");\n}',
+  t('a {\nfont:12px \'Helvetica\',"Arial",\'Nautica\';\nbackground:url("/some/nice/picture.png");\n}',
```

Comment:
what do you mean? My understanding is - on Windows you just can't continue the line like this with a backslash in the end of line. i.e. it is a functional fault of js parser we use on windows and not tool.